test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -455,6 +514,13 @@ instance ToSchema WalletPutData where
 instance ToSchema WalletPutPassphraseData where
     declareNamedSchema _ = declareSchemaForDefinition "WalletPutPassphraseData"
 
+instance ToSchema PostTransactionData where
+    declareNamedSchema _ = declareSchemaForDefinition "PostTransactionData"
+
+instance ToSchema ApiTransaction where
+    declareNamedSchema _ = declareSchemaForDefinition "Transaction"
```

Comment:
I was following similar naming scheme as other used in https://github.com/input-output-hk/cardano-wallet/blob/5c7d690e5d994c931e5b437c5bdbaf222f6dc675/src/Cardano/Wallet/Api.hs . 

I have to admit that I don't like these renamings here. I would rather be consistent and refactor swagger.json to follow haskell datatypes naming scheme or vice versa then to use this naming renaming hack. Grepping and other search tools fail due to this mechanism and developer explicitly have to be aware of this in order to follow the link in swagger.json.

Newcomers can't easily link haskell datatypes with swagger without figuring out that somewhere in code we do this renaming. My suggestion is to use same naming for swagger objects that we use for haskell datatypes. In the example above that would mean that I would rename object in `swagger.json` from "&Transaction" to "&ApiTransaction". I don't see any downsides (as this information is hidden in UI) and pros are that it would be much easier to link things together without knowing about this renaming rules

Reply:
There's no particular renaming "hack" here, it's just that names don't match. I agree it would be nice to have them match though (TODO in another PR though).